paasta_tools/cli/schemas/tron_schema.json

```diff
@@ -231,6 +231,9 @@
                         }
                     }
                 },
+                "uses_bulkdata": {
+                    "type": "boolean"
+                },
```

Comment:
oh, interesting - this was already at the right level - how did the schema not complain before?

Reply:
hmm, or maybe this is just the GH diff view lying to me - `paasta validate` doesn't fail with the current version, but with my branch it does

Reply:
I think it's the right indentation level but under the `job` schema definition rather than the `action` definition, so this PR does the right thing!